### PR TITLE
Add raw documents documentation

### PR DESCRIPTION
**Pull request by aeng**
Moved pages from 
https://github.com/zanata/zanata-server/wiki/Raw-Documents and https://github.com/zanata/zanata-server/wiki/Features to readthedocs.

### CODE REVIEW

File: docs/user-guide/documents/download-translated-documents.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Download translations
+# Download translated documents
 
 Zanata allow anyone to download translations from a project version.
```
Comment (djansen-redhat) [inline docs/user-guide/documents/download-translated-documents.md]: *allows

File: docs/user-guide/documents/download-translated-documents.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Zanata allow anyone to download translations from a project version.
 1. Go to project version page
 1. In Language tab, select a translation language that you wish to download.
 
-#### Download all translation documents
+#### Download all translated documents
 
 1. On right panel, click on the drop down menu, select `Download All(zip)`.
```
Comment (djansen-redhat) [inline docs/user-guide/documents/download-translated-documents.md]: On the right panel, click on the drop down menu and select

File: docs/user-guide/documents/download-translated-documents.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Zanata allow anyone to download translations from a project version.
 </figure>
 <br/>
 
-#### Download individual translation document
+#### Download individual translated document
 
 1. Click on drop down menu located in front of a document.
```
Comment (djansen-redhat) [inline docs/user-guide/documents/download-translated-documents.md]: Click on the

File: docs/user-guide/documents/raw-documents.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+_Open Document Formula (*.odf) and Open Document Database (*.odb) are not supported._
+
+## Known Issues
+
+This is an experimental feature, and several issues exist that should be considered when using this feature:
+
+### Translation Misalignment
+
+Uploading modified versions of existing documents can cause translation misalignment.
```
Comment (djansen-redhat) [inline docs/user-guide/documents/raw-documents.md]: Is this still valid? Did the shift from position to resource hash fix this?

File: docs/user-guide/documents/upload-documents.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Upload Strings
+# Upload documents
 
 Anyone with an account on Zanata can create a translation project for their strings.
```
Comment (djansen-redhat) [inline docs/user-guide/documents/upload-documents.md]: for their *documents*?

File: docs/user-guide/system-admin/configuration/installation.md [hunk 6]
```diff
@@ -106,4 +106,4 @@ Go to the `<JBOSS>/bin` directory and run the `standalone.sh` (Linux, Mac) or `s
 
 To start using your Zanata server, open a browser and navigate to `http://localhost:8080/zanata`
 
-You can now upload some source strings and start translating. To get started, see [Adding Source Strings](user-guide/projects/upload-strings).
+You can now upload some source strings and start translating. To get started, see [Adding Source Strings](user-guide/documents/upload-documents).
```
Comment (djansen-redhat) [inline docs/user-guide/system-admin/configuration/installation.md]: upload some source *documents*?

File: docs/user-guide/documents/raw-documents.md (hunk 4 above)
Comment (aeng) [inline docs/user-guide/documents/raw-documents.md]: You are right.

**Comment by djansen-redhat** [comment]
:+1:

**Pull request merged**